dojo/listbox.m.css

```diff
@@ -36,5 +37,16 @@
 }
 
 .selectedOption {
+	composes: checkIcon;
 	background-color: var(--color-background-faded);
+	padding-right: calc(var(--grid-base) * 4);
+	position: relative;
+}
+
+.selectedOption::before {
+	font-family: 'dojo2BaseTheme';
```

Comment:
what is `dojo2BaseTheme` ?

Reply:
The icon font name. Should probably be moved to a variable.